packages/core/src/codewhisperer/models/model.ts

```diff
@@ -330,17 +330,19 @@ export class ZipManifest {
     hilCapabilities: string[] = ['HIL_1pDependency_VersionUpgrade']
     transformCapabilities: string[] = ['EXPLAINABILITY_V1'] // TO-DO: for SQL conversions, maybe make this = []
     customBuildCommand: string = 'clean test'
-    requestedConversions: {
-        sqlConversion:
-            | {
-                  source: string | undefined
-                  target: string | undefined
-                  schema: string | undefined
-                  host: string | undefined
-                  sctFileName: string | undefined
-              }
-            | undefined
-    } = { sqlConversion: undefined }
+    requestedConversions:
+        | {
+              sqlConversion:
+                  | {
+                        source: string | undefined
+                        target: string | undefined
+                        schema: string | undefined
+                        host: string | undefined
+                        sctFileName: string | undefined
+                    }
+                  | undefined
+          }
+        | undefined = undefined
```

Comment:
This is the main change. Instead of defaulting to `{ sqlConversion: undefined }`, which puts `requestedConversions` in the `manifest.json`, default to `undefined`, which leaves `requestedConversions` out of the `manifest.json` entirely.